lib/matplotlib/axes/_axes.py

```diff
@@ -5178,97 +5178,114 @@ def pcolor(self, *args, **kwargs):
+            If `None`, default to rc settings.
 
-          *norm*: [ *None* | Normalize ]
-            An :class:`matplotlib.colors.Normalize` instance is used
-            to scale luminance data to 0,1. If *None*, defaults to
-            :func:`normalize`.
+        norm :`matplotlib.colors.Normalize`, optional, default: None
+            An instance is used to scale luminance data to (0, 1).
+            If `None`, defaults to :func:`normalize`.
 
-          *vmin*/*vmax*: [ *None* | scalar ]
-            *vmin* and *vmax* are used in conjunction with *norm* to
-            normalize luminance data.  If either is *None*, it
+        vmin, vmax : scalar, optional, default: None
+            `vmin` and `vmax` are used in conjunction with `norm` to
+            normalize luminance data.  If either is `None`, it
             is autoscaled to the respective min or max
-            of the color array *C*.  If not *None*, *vmin* or
-            *vmax* passed in here override any pre-existing values
-            supplied in the *norm* instance.
+            of the color array `C`.  If not `None`, `vmin` or
+            `vmax` passed in here override any pre-existing values
+            supplied in the `norm` instance.
 
-          *shading*: [ 'flat' | 'faceted' ]
+        shading : {'flat', 'faceted'}, optional, default: 'flat'
             If 'faceted', a black grid is drawn around each rectangle; if
             'flat', edges are not drawn. Default is 'flat', contrary to
             MATLAB.
 
             This kwarg is deprecated; please use 'edgecolors' instead:
-              * shading='flat' -- edgecolors='none'
-              * shading='faceted  -- edgecolors='k'
-
-          *edgecolors*: [ *None* | ``'none'`` | color | color sequence]
-            If *None*, the rc setting is used by default.
+              `shading`='flat' -- `edgecolors`='none'
+              `shading`='faceted'  -- `edgecolors`='k'
 
+        edgecolors : {None, ``'none'``, color, color sequence}
+            If `None`, the rc setting is used by default.
             If ``'none'``, edges will not be visible.
-
             An mpl color or sequence of colors will set the edge color
 
-          *alpha*: ``0 <= scalar <= 1``   or *None*
-            the alpha blending value
+        alpha : scalar, optional, default: None
+            The alpha blending value, between 0 (transparent) and 1 (opaque).
 
-          *snap*: bool
+        snap : bool, optional, default: False
             Whether to snap the mesh to pixel boundaries.
 
-        Return value is a :class:`matplotlib.collections.Collection`
-        instance.
+        Returns
+        -------
+        collection : `matplotlib.collections.Collection`
+
+        Other Parameters
+        -------
+        kwargs : `~matplotlib.collections.PolyCollection` properties:
+
+        %(PolyCollection)s
+
+        The default `antialiaseds` is False if the default
```

Comment:
Hi @NelleV , thanks for suggestions. I've been testing this doc a bit on a private branch, but had problems with understanding the problem. Did you mean removing the  line 5259 with `%(PolyCollection)s`. I did it, but didn't help and I saw it in other parts of the file. Will try to test it more during weekend, but if you have other suggestion, I'm happy to try.